container/es-releases.json

```diff
@@ -1,16 +1,25 @@
 {
   "release": {
-       "2.0.1.0" :{
+       "2.0.1.0" : {
           "sparkversion": "2.4.6",
            "sparkclient": "repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar",
            "hadoopversion":"2.6",
-           "scopt211version":"3.5.0"
+           "scopt211version":"3.5.0",
+           "kafkatag":"v2.0.1.0"
           } ,
        "2.0.0.5" : {
           "sparkversion": "2.2.1",
            "sparkclient": "repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.2.1/2.0.0/ibm-db2-eventstore-client-spark-2.2.1-2.0.0.jar",
            "hadoopversion":"2.6",
-           "scopt211version":"3.5.0"
+           "scopt211version":"3.5.0",
+           "kafkatag":"v2.0.0.5"
+          } ,
+       "dev" : {
```

Comment:
What's the idea behind this ?  Is it just a placeholder to make it easier for devs to make changes going forward ?